### DISCUSSION TITLE
Bot在搬运issue时同时在issue下面留出对应discussion的链接

### DISCUSSION THREAD

**Discussion by MKStoler4096** [ClassIsland/ClassIsland#374] [常规]
Bot在搬运issue时n能不能同时在issue下面留出对应discussion的链接（）

**Comment by LyCecilion** [comment]
同求！刚想发个 Discussions 说这事（）

或许只需要在 voting issue 中 refer 到源 issue 就行了，这样在源 issue 下方就会出现 `classisland-bot mentioned this issue x ago`

**Reply by MKStoler4096** [reply]
对的（）

**Comment by HelloWRC** [comment]
已实现。

> 或许只需要在 voting issue 中 refer 到源 issue 就行了

其实每次搬运都 refer 了，但好像在 discussion 里引用 issue 不会在源 issue 显示引用（？），也可能是我引用的姿势不对

**Reply by LyCecilion** [reply]
> 已实现。
> 
> > 或许只需要在 voting issue 中 refer 到源 issue 就行了
> 
> 其实每次搬运都 refer 了，但好像在 discussion 里引用 issue 不会在源 issue 显示引用（？），也可能是我引用的姿势不对

我也注意到这个问题了，有的出现了有的没出现🤔

**Comment by HelloWRC** [comment]
#1
